src/web/WebSchedulerService.cpp

```diff
@@ -37,14 +37,14 @@ void WebSchedulerService::begin() {
 void WebScheduler::read(WebScheduler & webScheduler, JsonObject & root) {
     JsonArray schedule = root.createNestedArray("schedule");
     for (const ScheduleItem & scheduleItem : webScheduler.scheduleItems) {
-        JsonObject si     = schedule.createNestedObject();
-        si["id"]          = scheduleItem.id; // name, is unqiue
-        si["active"]      = scheduleItem.active;
-        si["flags"]       = scheduleItem.flags;
-        si["time"]        = scheduleItem.time;
-        si["cmd"]         = scheduleItem.cmd;
-        si["value"]       = scheduleItem.value;
-        si["description"] = scheduleItem.description;
+        JsonObject si = schedule.createNestedObject();
+        si["id"]      = scheduleItem.id; // name, is unqiue
```

Comment:
The id is undefined here, go back to `smallitoa(counter)`, or skip id here and generate all unique ids in webpage on load.

Reply:
yes, that's next to do. see the previous comment

Reply:
fixed.